.github/workflows/publish-utils.yml

```diff
@@ -2,7 +2,7 @@ name: Publish wavs-wasi-utils to crates.io
 on:
   push:
     tags:
-      - 'v*'
+      - '[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
the [github docs on the releases page](https://github.com/Lay3rLabs/wavs-wasi/releases/new) says:

> It’s common practice to prefix your version names with the letter v. Some good tag names might be v1.0.0 or v2.3.4.

is there a compelling reason to _not_ do that?

Reply:
I was just thinking of how to avoid tags not triggering CI because of forgetting to prefix with v, and I thought it was slightly cleaner. If it's recommended, then let's just continue with it then.